packages/sui-mono/bin/sui-mono-release.js

```diff
@@ -38,7 +38,7 @@ const releaseEachPkg = ({pkg, code} = {}) => {
       ['git', ['add', cwd]],
       ['sh', ['-c', `VERSION=$(node -p -e "require('./package.json').version") && git commit -m "release(${pkg}): v$VERSION"`]],
       ['npm', ['publish', `--access=${publishAccess}`]],
-      ['git', ['push', 'origin', 'master']]
+      ['git', ['push', 'origin', 'HEAD']]
     ]
     scripts['build'] && commands.unshift(['npm', ['run', 'build']])
```

Comment:
Now that I'm checking this.

and the migration of the optimizely studio, do you need at all the task `version: npm run build` in the component scripts?